napari/layers/utils/plane_manager.py

```diff
@@ -0,0 +1,75 @@
+from typing import Tuple
+
+import numpy as np
+from pydantic import validator
+
+from ...utils.events import EventedModel
+from ...utils.geometry import intersect_line_with_plane_3d
+from ...utils.misc import ensure_n_tuple
+
+
+class PlaneManager(EventedModel):
+    """Manages properties relating to plane rendering in 3D.
+
+    In this object, planes in 3D are defined by a position, a normal vector
+    and a thickness parameter.
+
+    Attributes
+    ----------
+    position : 3-tuple
+        A 3D position on the plane, defined in data coordinates.
+    normal : 3-tuple
+        A 3D unit vector normal to the plane, defined in data coordinates.
+    thickness : float
+        Thickness of the slice
+    enabled : bool
+        Whether plane rendering is enabled.
+    """
+
+    position: Tuple[float, float, float] = (0, 0, 0)
+    normal: Tuple[float, float, float] = (1, 0, 0)
+    thickness: float = 10.0
+    enabled: bool = False
+
+    @validator('position', 'normal', pre=True)
```

Comment:
have you double checked that this is necessary?  I think pydantic should do this when you annotate as `Tuple[x, x, x]`